src/Tribe/PUE/Checker.php

```diff
@@ -568,6 +571,8 @@ public function validate_key( $key ) {
 					}
 				}
 
+				$pue_notices->clear_notices( $plugin_name );
```

Comment:
When a valid license key is entered and gets ajax verified, it actually _does_ get saved. The only time the ajax verification doesn't get saved is when the key is invalid, requires an upgrade, or expired.